airflow/providers/snowflake/example_dags/example_snowflake.py

```diff
@@ -55,6 +57,22 @@
     dag=dag,
 )
 
+slack_report = SnowflakeToSlackOperator(
+    task_id="slack_report",
+    sql="""
+    SELECT O_ORDERKEY, O_CUSTKEY, O_ORDERSTATUS
+    FROM {0}
+    LIMIT 10;
+    """.format(SNOWFLAKE_SAMPLE_TABLE),
+    slack_message="""
+    Results in an ASCII table:
+    ```{{ results_df | tabulate(tablefmt="pretty", headers="keys") }}```
+    """,
+    snowflake_conn_id=SNOWFLAKE_CONN_ID,
+    slack_conn_id=SLACK_CONN_ID,
+    dag=dag
+)
```

Comment:
What would you say to extract `sql` and `slack_message` to variables for easier reading? We can also use f-strings :)